src/code_base/lognormal_GAMMA_efsa_NC.cpp

```diff
@@ -443,7 +450,7 @@ std::vector<double> lognormalGAMMA_efsa_BMD_NC::bmd_start_hybrid_extra_clean(std
 	double TAIL_PROB = tail_prob;
 	Eigen::MatrixXd theta2(x.size(), 1);
 	/////////////////////////////////////////////////////////////////////////////////////
-	for (int i = 0; i < x.size(); i++)
+	for (size_t i = 0; i < x.size(); i++)
```

Comment:
Compiler warning about comparing 2 different types. 
Mainly the changes are:
`size_t` and `unsigned`

See the change a few lines above. similar changes across all model files.